packages/sku/src/services/telemetry/metricsMeasurers.ts

```diff
@@ -0,0 +1,42 @@
+import debug from 'debug';
+
+const log = debug('sku:metrics');
+
+const skuStartMarkName = 'skuStartComplete';
+const skuStart = {
```

Comment:
This module is an attempt to unify our metric measurements into a single place. It might even be worth emitting telemetry from here as well, so that the raw `provider` API is abstracted away into something more type-safe and explicit.

The distinction I've made between `metrics` and `telemetry` is that the metrics are the actual measurements and values, whereas the telemetry is the communication of those metrics to somewhere.